config/i18n-tasks.yml

```diff
@@ -44,13 +46,20 @@ data:
 search:
   ## Paths or `File.find` patterns to search in:
   paths:
+      <%= "- #{%x[bundle show devise-i18n].chomp}/app\n" %>
```

Comment:
What about:

```ruby
Gem.loaded_specs['devise-i18n'].full_gem_path
```

Reply:
Running `i18n-tasks health en` fails:

    undefined method `full_gem_path` for nil:NilClass

Running it with `bundle exec` makes it work.